types/prosemirror-model/index.d.ts

```diff
@@ -1426,7 +1426,7 @@ export class DOMSerializer<S extends Schema = any> {
      * document, should be passed so that the serializer can create
      * nodes.
      */
-    serializeFragment(fragment: Fragment<S>, options?: { [key: string]: any }): DocumentFragment;
+    serializeFragment(fragment: Fragment<S>, options?: { [key: string]: any }, target?: ProsemirrorNode<S>): DocumentFragment;
```

Comment:
It looks like `target` should be a [DocumentFragment](https://prosemirror.net/docs/ref/#model.DOMSerializer.serializeFragment), but it isn't part of the public API. See https://prosemirror.net/docs/ref/#model.DOMSerializer.serializeFragment